### PR TITLE
Feat/support zkm builder

### PR DESCRIPTION
**Pull request by xander42280**

### CODE REVIEW

File: host-program/sha2-rust/src/main.rs [hunk 1]
```diff
@@ -8,6 +8,8 @@ use zkm_sdk::{prover::ClientCfg, prover::ProverInput, ProverClient};
 
 pub const DEFAULT_PROVER_NETWORK_RPC: &str = "https://152.32.186.45:20002";
 pub const DEFALUT_PROVER_NETWORK_DOMAIN: &str = "stage";
+const ELF_PATH: &str =
+    "../guest-program/sha2-rust/target/elf-compilation/mips-unknown-linux-musl/release/sha2-rust";
```
Comment (eigmax) [inline host-program/sha2-rust/src/main.rs:12]: try to reuse the env variable CARGO_TARGET_DIR to find the target. https://doc.rust-lang.org/cargo/reference/environment-variables.html

Comment (xander42280) [reply host-program/sha2-rust/src/main.rs]: Updated, [https://github.com/zkMIPS/zkm-project-template/pull/41/commits/c17e28acdfbd9188f16aa26d173f75a19a4e7dfc](url)

File: host-program/revme/src/main.rs [hunk 2]
```diff
@@ -7,6 +7,10 @@ use zkm_sdk::{prover::ClientCfg, prover::ProverInput, ProverClient};
 
 pub const DEFAULT_PROVER_NETWORK_RPC: &str = "https://152.32.186.45:20002";
 pub const DEFALUT_PROVER_NETWORK_DOMAIN: &str = "stage";
+const ELF_PATH: &str = concat!(
+    env!("CARGO_MANIFEST_DIR"),
+    "/../../guest-program/revme/elf/mips-unknown-linux-musl"
```
Comment (eigmax) [inline host-program/revme/src/main.rs:12]: It's better to use a global variable to represent the target mips-unknown-linux-musl.

Comment (xander42280) [reply host-program/revme/src/main.rs:12]: Okex

**Review by eigmax — APPROVED** [review]

**Pull request merged**